lib/gui/lvgl/Kconfig

```diff
@@ -21,6 +22,14 @@ config APP_LINK_WITH_LVGL
 	  disabled if the include paths for LVGL are causing aliasing
 	  issues for 'app'.
 
+config LVGL_USER_CONFIG_FILE
+    bool "Use user config file"
+    default n
+
+config LVGL_USER_CONFIG_FILE_INCLUDE
```

Comment:
Suggestion: `LVGL_USER_CONFIG_FILE_NAME` (or, if we decide to change the config name above, something equivalent) would make it more clear as to what this config is supposed to describe.

Reply:
it would probably be correct to name LVGL_USER_CONFIG_FILE, since the full name of the include file is stored in this variable

Reply:
If `LVGL_USER_CONFIG_FILE` is renamed to `LVGL_USE_USER_CONFIG`, sure.